libs/platform/src/lib/components/form/data-model.ts

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
I though that this one is under: 

https://github.com/SAP/fundamental-ngx/blob/master/libs/platform/src/lib/domain/data-model.ts

You have moved this?